website/app/styles/pages/home.scss

```diff
@@ -4,7 +4,14 @@
   grid-area: sidebar / sidebar / content / content;
 }
 
+.doc-page-home__hero {
+  padding-bottom: 80px;
```

Comment:
I'm not following how this math worked out (maybe we can pair on this next week?), but something is a bit off with the height of the cut-out in the hero...it's a little too tall and should only overlap the bottom of the hero by 52px.

Reply:
it's a super-preliminary layout. I am pretty sure will change anyway, it was mainly to mimic (more or less) what you had in Figma, so definitely not pixel perfect (not by a mile)